src/test/java/org/assertj/core/api/junit/jupiter/BDDSoftAssertionsExtensionIntegrationTest.java

```diff
@@ -40,7 +41,7 @@
  * @since 3.13
```

Comment:
Few rows above there is probably a `@link` typo: `SoftAssertionsExtensionTest` instead of `SoftAssertionsExtensionIntegrationTest`?

Reply:
Yep, good catch! I'll fix that.